librz/core/cmd_zign.c

```diff
@@ -1017,7 +1017,7 @@ static bool do_bestmatch_sig(RzCore *core, int count) {
 		RzSignBytes *b = item->bytes;
 		int minsz = rz_config_get_i(core->config, "zign.minsz");
 		if (b && b->size < minsz) {
-			eprintf("Warning: Function signature is too small (%d < %d) See e zign.minsz", b->size, minsz);
+			eprintf("Warning: Function signature is too small (%d < %d) See e zign.minsz \n", b->size, minsz);
 			rz_sign_item_free(item);
 			return false;
 		}
```

Comment:
Since it is just a warning instead of an error, we can also delete the following two lines to show result.

Reply:
```asm
[0x1400033cc]> pdf
        ╎   ; CALL XREF from main @ 0x140001033
        ╎   ;-- fcn.1400033cc:
┌ sym.___common_fsopen_D__YAPEAU_iobuf__QEBD0H_Z (uint32_t arg1, uint32_t arg2);
│ bp: 0 (vars 0, args 0)
│ sp: 4 (vars 4, args 0)
│ rg: 2 (vars 0, args 2)
│       ╎   0x1400033cc      mov   r8d, 0x40                           ; '@' ; 64
│       └─< 0x1400033d2      jmp   sign.bytes.sym.___common_fsopen_D__YAPEAU_iobuf__QEBD0H_Z_0
[0x1400033cc]> zb
Warning: Function signature is too small (11 < 16) See e zign.minsz 
1.00000 B   sym._wfopen
1.00000 B   sym.fopen
0.72727 B   sym._maximum_signed_value___crt_strtox__YA_J_K_Z
```
It is strange that `zb` gives `sym._wfopen` and `sym.fopen` as the best match zignature at `0x1400033cc` but `zo`+`aaa` gives wrong anwser.